leaderboard/queries/query.py

```diff
@@ -207,3 +207,13 @@
   }
 }
 """
+
+user_query = """
+query userQuery($username: String!) {
+   user(login: $username) {
```

Comment:
@shresthalucky My first thought was we had to query the avatarUrl for the user but we don't need to do so. You can get the user avatar from username directly.

https://github.com/shresthalucky.png?size=20

So you can just add `.png?size=20` to the github profile URL.

This reduces all the extra queries and requests and is much easier to do.

Reply:
Thank you for cool suggestion. Made changes to PR.